packages/glibc_build232.rb

```diff
@@ -332,7 +332,7 @@ def self.postinstall
       end
     end
     puts 'Updating glibc package filelist...'.lightblue
-    File.open("#{CREW_META_PATH}/glibc_build.filelist", 'w+') do |f|
+    File.open("#{CREW_META_PATH}/glibc_build232.filelist", 'w+') do |f|
```

Comment:
Add new manifest files and remove the old?

Reply:
we already have the correctly named manifests, the package just wasn't looking for them.